mods/screwdriver/init.lua

```diff
@@ -80,6 +130,7 @@ screwdriver.handler = function(itemstack, user, pointed_thing, mode, uses)
 	if should_rotate then
 		node.param2 = new_param2
 		minetest.swap_node(pos, node)
+		minetest.check_for_falling(pos)
```

Comment:
Sorry i mean, how about this line is only run for wallmounted nodes. It's best to avoid falling checks when not needed (for facedir).